src/features/Notifications/Home/_components/DailyDigest.tsx

```diff
@@ -133,6 +133,13 @@ const DailyDigest = ({
                       })}
                     </li>
                   )}
+                {planSections?.dataExchangeApproachMarkedComplete && (
+                  <li className="line-height-sans-5">
+                    {notificationsT(
+                      'index.dailyDigest.dataExchangeApproachComplete'
+                    )}
+                  </li>
+                )}
```

Comment:
Minor `nit` -- but would you mind adding the `planSections.dataExchangeApproachMarkedComplete` property to the mocks for this test and re-running snapshots and/or adding an assertion that the bullet shows up in the change?